### ISSUE TITLE
Pre-Rotation of co-ordinates

### ISSUE THREAD

**Issue by steven-murray** [telegraphic/PyGSM#6]
I was wondering if it is considered a good idea to have the ability to pre-rotate the co-ordinates of the map? 

If one requires output at many frequencies, and all of them to be rotated, then it is particularly inefficient to rotate the map at each frequency. However, if the 6 underlying maps are each rotated to begin with before interpolation, it is much more efficient. I have code that does this which I am currently using (and it seems to work quite well for my specific use-case). I can make a PR if you think it's something that would be useful.

**Comment by telegraphic** [comment]
Agreed that this may be more efficient for many use cases. If you are happy to try a PR please do go ahead.

I don't have much/any time over the next few months, but would you be interested in a 'hackathon' day later in the year to work on this + other features?

**Comment by steven-murray** [comment]
Sorry for the delay... I've sent a pull request that makes these changes: https://github.com/telegraphic/PyGSM/pull/8

Let me know what you think. 

I've been pretty busy for the latter half of the year... if you have some stuff you'd like to work on, let me know and I can do a day maybe towards the start of next year?
